### DISCUSSION TITLE
Updating to 10.0 from 9.0.

### DISCUSSION THREAD

**Discussion by geertjanknapen1** [laravel/framework#47151] [Q&A] [answered]
So we are updating from L9 to L10, which is not going smoothly. There are no issues with 3rd party packages anymore, I resolved those, the only issue I'm still running into is;

> Only one of these can be installed: laravel/framework[v10.0.0, ..., 10.x-dev], illuminate/contracts[v8.0.0, ..., 8.x-dev, v9.0.0-beta.1, ..., 9.x-dev, v10.0.0, ..., 10.x-dev]. laravel/framework replaces illuminate/contracts and thus cannot coexist with it.

`Illuminate/contracts` seems to be a package used internally by Laravel, since I don't see it in my `composer.json`.

---

Any help as to why this is happening is appreciated.

```json
"require": {
        "php": "^8.1",
        "ext-intl": "*",
        "aryehraber/statamic-impersonator": "^2.4",
        "doctrine/dbal": "^3.6",
        "drewm/mailchimp-api": "^2.5",
        "guzzlehttp/guzzle": "^7.5",
        "laravel/framework": "^10.0",
        "laravel/sanctum": "^3.2",
        "laravel/socialite": "^5.6",
        "laravel/tinker": "^2.8",
        "laravelcollective/html": "^6.4",
        "sinergi/browser-detector": "^6.1",
        "socialiteproviders/microsoft-azure": "^5.1",
        "spatie/regex": "^3.1",
        "spatie/statamic-responsive-images": "^4.0",
        "statamic/cms": "^4.0",
        "statamic/eloquent-driver": "^2.0",
        "therobfonz/laravel-mandrill-driver": "^4.0",
        "withcandour/aardvark-seo": "^2.0"
    },
    "require-dev": {
        "arcanedev/laravel-lang": "^10.0",
        "barryvdh/laravel-debugbar": "^3.8",
        "barryvdh/laravel-ide-helper": "^2.13",
        "cweagans/composer-patches": "^1.7",
        "fakerphp/faker": "^1.21",
        "laravel/pint": "^1.9",
        "laravel/sail": "^1.21",
        "mockery/mockery": "^1.5",
        "nunomaduro/collision": "^6.4",
        "phpunit/phpunit": "^9.6",
        "spatie/laravel-ignition": "^2.0",
        "symplify/vendor-patches": "^11.2"
    },
```

**Comment by rojtjo** [comment, marked as answer]
It seems that two of the packages you require do not have support for Laravel 10 yet:

- [herobfonz/laravel-mandrill-driver](https://packagist.org/packages/therobfonz/laravel-mandrill-driver#4.0.2)
Version 5 seems to have support for Laravel 10
`"therobfonz/laravel-mandrill-driver": "^5.0",`

- [arcanedev/laravel-lang](https://packagist.org/packages/arcanedev/laravel-lang#10.0.0)
Version 11 seems to have support for Laravel 10, but hasn't been released yet
`"arcanedev/laravel-lang": "11.x-dev",`

**Reply by geertjanknapen1** [reply]
@rojtjo Thanks Roy, seems I jumped the gun on this. arcanedev package was indeed throwing errors but they weren't regarding `illuminate/contracts` or laravel 10 so I figured it was not causing issues. 

It's fixed now, thanks for the reply anyways!
